airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 69589781-7828-43c5-9f63-8925b1c1ccc2
-  dockerImageTag: 4.5.10
+  dockerImageTag: 4.5.11
```

Comment:
You need to update the docs changelog too if not tests will fail. Also Can you create a separated branch than the master? It make easier to run tests when I duplicated your PR.

Reply:
👍

Reply:
I re-created the PR on a different base branch at https://github.com/airbytehq/airbyte/pull/36160